uber/templates/base.html

```diff
@@ -112,7 +112,7 @@
     {% block head_additional %}{% endblock %}
 </head>
 <body>
-<div class="loader"><a class="loader_link" href="../static_views/slow_load.html" target="_blank"></a></div>
+{% if not OFFLINE_MODE and not COLLECT_INTERESTS %}<div class="loader"><a class="loader_link" href="../static_views/slow_load.html" target="_blank"></a></div>{% endif %}
```

Comment:
`COLLECT_INTERESTS` is defunct, but the slow-load page should still be disabled in offline mode.